plugins/woocommerce-blocks/tests/e2e/tests/cart/cart-checkout-block-shipping.block_theme.spec.ts

```diff
@@ -58,9 +58,11 @@ test.describe( 'Shopper → Shipping', () => {
 		await userFrontendUtils.addToCart( REGULAR_PRICED_PRODUCT_NAME );
 		await userFrontendUtils.goToCart();
 
-		await expect(
-			userPage.getByLabel( 'Add an address for shipping options' )
-		).toBeVisible();
+		const button = userPage.locator( 'button', {
+			hasText: 'Enter address to check delivery options',
+		} );
+
+		await expect( button ).toBeVisible();
```

Comment:
Why not keep this as one statement? I'd break out `const button` if the button is reused, which in this case, it isn't.

```suggestion
		await expect( 
				userPage.locator( 'button', { hasText: 'Enter address to check delivery options' } )
		 ).toBeVisible();
```